apps/health_alerts/models.py

```diff
@@ -108,3 +112,47 @@ def get_priority_color(self):
 
     def serve(self, request):
         return redirect(self.alert_file.url)
+
+
+class HealthAlertsSignUp(models.Model):
+    """Health Alert Sign Up
+
+    This model is a standard django model used
```

Comment:
I would amend to mention something like "Those newsletters are managed completely outside our system."